packages/shared/src/types/abTests/header.ts

```diff
@@ -21,5 +21,4 @@ export interface HeaderTestSelection {
     test: HeaderTest;
     variant: HeaderVariant;
     modulePathBuilder: (version?: string) => string;
-    moduleName: string;
```

Comment:
the `moduleName` field wasn't being used (the name is hardcoded in `payloads.ts`, which is fine) so I've removed it here